src/ComponentPage.js

```diff
@@ -21,7 +23,7 @@ class ComponentPage extends Component {
   renderSidebar(activeLink) {
     const links = [{
       content: 'Button',
-      url: '/button',
+      url: `/button${pageExt}`,
```

Comment:
If we put the ${pageExt} in the `<a>` in the `renderSidebarLink` method, we wouldn't have to reference pageExt as many times. Although it could cause confusion.